components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -78,22 +112,36 @@ export default class ChannelIntroMessage extends React.PureComponent<Props> {
             centeredIntro = 'channel-intro--centered';
         }
 
+        let classes: string;
+        const createClasses = (withSvg: boolean) => {
+            if (this.props.theme?.type === 'Onyx' && !withSvg) {
```

Comment:
since the `theme` variable is only used for this and it is not really needed for this condition we can just remove it from the component